src/main/java/com/example/heboard/global/config/SecurityConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.example.heboard.global.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
+
+    @Bean
```

Comment:
jwt 기반으로 로그인을 구현하실 거면

```suggestion
    .formLogin().disable()
              .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
```
코드 추가하셔도 좋을 거 같아요